.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
+2.
+3.
+4.
+
+<details><summary>lerna.json</summary><p>
+<!-- browsers demand the next line be empty -->
+
+```json
+<!-- Please paste your `lerna.json` here -->
+```
+</p></details>
+
+<details><summary>lerna-debug.log</summary><p>
+<!-- browsers demand the next line be empty -->
+
+```txt
+<!-- If you have a `lerna-debug.log` available, please paste it here -->
+<!-- Otherwise, feel free to delete this <details> block -->
+```
+</p></details>
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+| `lerna --version` |  |
```

Comment:
Yeah, I'm fiddling with it in a new issue right now. Seems GH tables _require_ a header row?